app/helpers/preferences_helper.rb

```diff
@@ -1,5 +1,12 @@
 # Helper methods for per-User preferences
 module PreferencesHelper
+  def layout_choices
+    [
+        ['Small', :small],
+        ['Wide', :wide]
```

Comment:
@gopeter how about name it `fixed` and `fluid`. I think it will better describe the setting